app/screens/settings/about/about.tsx

```diff
@@ -119,6 +121,18 @@ const About = ({componentId, config, license}: AboutProps) => {
     const intl = useIntl();
     const theme = useTheme();
     const styles = getStyleSheet(theme);
+    const serverUrl = useServerUrl();
+    const [loadMetric, setLoadMetric] = useState<number | null>(null);
+
+    useEffect(() => {
+        const fetchLoadMetric = async () => {
+            const isLicensed = license.IsLicensed === 'true';
+            const metric = await getLicenseLoadMetric(serverUrl, config.Version, isLicensed);
+            setLoadMetric(metric);
+        };
+
+        fetchLoadMetric();
+    }, [config.Version, license.IsLicensed, serverUrl]);
```

Comment:
I wonder if it would be best to move this to a LoadMetric component that is only mounted if isLicense === 'true' to avoid re-rendering the whole screen, thoughts?

Reply:
I'm inclined to keep it inline, but I'm also not familiar with the cost of a re-render for a mostly static screen. Is the impact high as such?

Reply:
Is not as high in this particular case to be fair.. is ok to leave as is and in the event of this screen gets a but more complex the refactor can always be done